package.json

```diff
@@ -9,15 +9,29 @@
   },
   "license": "MIT",
   "author": "Neolution",
+  "sideEffects": false,
+  "exports": {
+    "./package.json": "./package.json",
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.modern.js",
+      "require": "./dist/index.cjs"
+    }
+  },
   "main": "dist/index.js",
+  "umd:main": "dist/index.umd.js",
+  "jsdelivr": "dist/index.umd.js",
+  "unpkg": "dist/index.umd.js",
   "module": "dist/index.modern.js",
   "source": "src/index.ts",
+  "jsnext:main": "dist/index.modern.js",
+  "types": "dist/index.d.ts",
   "files": [
     "dist"
   ],
   "scripts": {
-    "build": "shx rm -rf dist && microbundle --jsx React.createElement --jsxFragment React.Fragment --jsxImportSource react --globals react/jsx-runtime=jsx --no-compress --tsconfig ./tsconfig.microbundle.json",
-    "build-storybook": "cross-env NODE_OPTIONS=--openssl-legacy-provider build-storybook",
+    "build": "rollup -c",
```

Comment:
@neotob 

In the start script we should also switch to rollup. `rollup -c -w`

Reply:
changed it, i wasnt sure if the start is used at all.. because im not sure what exactly should be started in a package 🤔

Reply:
@neotob 

i guess `start` is a bit missleading but i felt like it's common to either have `dev` or `start` as the main script.

What it does: It launches rollup in watch mode. So whenever you change a file it will get recompiled.